BREAKING.md

```diff
@@ -24,6 +24,7 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 `IPendingFlush` has been deprecated. Use batch metadata on `IPendingMessage` instead to indicate the end of a batch.
 
 ## 2.0.0-internal.3.0.0 Breaking changes
+- [Sub Directory creation with same name fail in Shared Directory DDS](#Sub-Directory-creation-with-same-name-fail-in-Shared-Directory-DDS)
```

Comment:
Maybe this should be rephrased to `Creating duplicate subdirectories in Shared Directory DDS will throw an error`